guides/common/modules/proc_updating-disconnected-server-on-EL9.adoc

```diff
@@ -0,0 +1,148 @@
+[id="Updating-Disconnected-{project-context}-on-EL9_{context}"]
+= Updating a disconnected {ProjectServer} on {EL} 9
+
+.Procedure on the connected {ProjectServer}
+. Ensure that you have synchronized the following repositories in your connected {ProjectServer}:
++
+* {RepoRHEL9BaseOS}
+* {RepoRHEL9AppStream}
+* {RepoRHEL9ServerSatelliteServerProjectVersion}
+* {RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+. Download the debug certificate of the organization and store it locally at `/etc/pki/katello/certs/org-debug-cert.pem` or a location of your choosing.
+For more information, see {AdministeringDocURL}Creating_an_Organization_Debug_Certificate_admin[Creating an Organization Debug Certificate] in _{AdministeringDocTitle}_.
+. Create a Yum configuration file under `/etc/yum.repos.d`, such as `_{project-context}-disconnected_.repo`, with the following contents:
++
+[options="nowrap" subs="attributes"]
+----
+[{RepoRHEL9BaseOS}]
+name=Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/rhel9/9/x86_64/baseos/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL9AppStream}]
+name=Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/rhel9/9/x86_64/appstream/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL9ServerSatelliteServerProjectVersion}]
+name={ProjectName} {ProjectVersion} for RHEL 9 RPMs x86_64
+baseurl=_https://{foreman-example-com}_/pulp/content/_My_Organization_/Library/content/dist/layered/rhel9/x86_64/satellite/{ProjectVersion}/os
+enabled=1
```

Comment:
```suggestion
enabled = 1
```

to algin with "sslverify" and all other lines.

Same applies to lines 20, 29, 47, 105, 110, 115, and 120.